portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase

```diff
@@ -74,6 +74,67 @@ definition {
 			value1 = "Banner Title Example");
 	}
 
+	@description = "Display correct experience for the logged user"
+	@priority = "5"
+	test DisplayUserExperience {
+		property portal.acceptance = "true";
+
+		Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/experience-content-page");
+
+		ContentPages.clickPencil();
+
+		PageEditor.addFragment(
+			collectionName = "Basic Components",
+			fragmentName = "Heading");
+
+		PageEditor.addExperience(experienceName = "exp1");
+
+		Refresh();
+
+		PageEditor.changeExperience(newExperience = "exp1");
+
+		AssertTextEquals(
+			locator1 = "Experience#FRAGMENT_HEADING",
+			value1 = "Heading Example");
+
+		PageEditor.editFragmentText(
+			fragmentName = "Heading",
+			id = "element-text",
+			text = "Edited Inline Text");
+
+		AssertTextEquals(
+			locator1 = "Experience#FRAGMENT_HEADING",
+			value1 = "Edited Inline Text");
+
+		PageEditor.changeExperience(newExperience = "Default");
+
+		AssertTextEquals(
+			locator1 = "Experience#FRAGMENT_HEADING",
+			value1 = "Heading Example");
+
+		Button.clickPublish();
+
+		Navigator.gotoSitePage(
+			pageName = "Experience Content Page",
+			siteName = "Test Site Name");
+
+		AssertTextEquals(
+			locator1 = "Experience#PUBLISHED_HEADING",
+			value1 = "Edited Inline Text");
+
+		User.logoutPG();
+
+		Navigator.gotoSitePage(
+			pageName = "Experience Content Page",
+			siteName = "Test Site Name");
+
+		AssertTextEquals(
+			locator1 = "Experience#PUBLISHED_HEADING",
+			value1 = "Heading Example");
+
+		User.firstLoginPG();
```

Comment:
why you have it?

Reply:
The Teardown breaks because of the test logs out